lib/ansible/modules/notification/cisco_spark.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+ANSIBLE_METADATA = {
+    'status': ['stableinterface'],
+    'supported_by': 'community',
+    'version': '1.0'
+}
+
+DOCUMENTATION = '''
+---
+module: cisco_spark
+short_description: Send a message to a Cisco Spark Room or Individual.
+description:
+    - Send a message to a Cisco Spark Room or Individual, with options to control the formatting.
+version_added: "1.0"
+author: Drew Rusell (@drusse11)
+notes:
+  - The receipient_id type must be valid for the supplied recipient_type.
```

Comment:
From the mail I got I saw this obvious typo "receipient_id".

Also now that we are at it, parameters can be indicated using C(parameter). So that line should read:
```yaml
- The C(recipient_id) type must be valid for the supplied C(recipient_type).
```